components/datetime/src/neo_marker.rs

```diff
@@ -296,39 +295,40 @@
 //! )
 //! .unwrap();
 //!
-//! // "uschi" - has metazone symbol data for generic_non_location_short
-//! let mut time_zone = "-0600".parse::<CustomTimeZone>().unwrap();
-//! time_zone.time_zone_id = mapper.as_borrowed().iana_to_bcp47("America/Chicago");
-//! time_zone.maybe_calculate_metazone(&mzc, &datetime);
+//! // "uschi" - has symbol data for generic_non_location_short
+//! let time_zone = TimeZoneInfo {
+//!     time_zone_id: mapper.as_borrowed().iana_to_bcp47("America/Chicago"),
+//!     offset: Some(UtcOffset::from_eighths_of_hour(-6 * 8)),
+//!     local_time: Some((datetime.date, datetime.time)),
```

Comment:
Thought: internally we always convert this to a local timestamp. Maybe we should take a `LocalTimestamp` (new type) instead of the weird tuple.

Reply:
A timestamp in local-time minutes is so odd, I don't want to expose that. This should stay a internal encoding of `(Date<Iso>, Time)`.

Reply:
If we exposed it, I would say to use an `i64` local-seconds timestamp, which is slightly less weird.

Reply:
The local thing is the big weirdness, not the minutes.